src/pages/events/project-setup.md

```diff
@@ -1,58 +1,179 @@
+
+   ```bash
+   cat config/private.key | pbcopy
+   ```
+
+1. Copy the contents of the `<workspace-name>.json` file into the **Adobe I/O Workspace Configuration** field.
+
+1. Enter a unique identifier in the **Adobe Commerce Instance ID** field. This value can be any string, but it must be unique.
+
+1. Click **Save Config** but do not leave the page.
+
+## Create an event provider and finish Commerce configuration
+
+You cannot create an event provider until after you have configured and saved the private key, workspace file, and instance ID values.
+
+1. Create a `<Commerce-root-directory>/app/etc/event-types.json` file and add the following contents:
+
+   ```json
+   {
+    "provider": {
+        "label": "My Adobe Commerce Events",
+        "description": "Provides out-of-process extensibility for Adobe Commerce"
+        }
+    }
+
+1. Run the following command to create an event provider:
+
+   ```bash
+   bin/magento events:create-event-provider
+   ```
+
+   The command displays a message similar to the the following:
+
+   ```terminal
+   No event provider found, a new event provider will be created
+   A new event provider has been created with ID 63a1f8fe-e911-45a4-9d3f
+   ```
+
+1. Copy the ID returned in the command output into the **Adobe I/O Event Provider ID** field in the Admin.
+
+1. Click **Save Config**.
+
+## Subscribe and register events
+
+You must define which Commerce events to subscribe to, then register them in the project.
+
+Commerce provides two sources for events: observers and plugins. You must specify the source as part of the event name. See [Subscribe to a Commerce event](./commands.md#subscribe-to-a-commerce-event) for details about the syntax of the `events:subscribe` command.
+
+1. Use the `events:subscribe` command to subscribe to Commerce events, as shown in the following examples:
+
+   ```bash
+   bin/magento events:subscribe observer.observer.catalog_product_save_after
+   ```
+
+   ```bash
+   bin/magento events:subscribe observer.customer_login
+   ```
+
+   ```bash
+   bin/magento events:subscribe plugin.magento.customer.api.account_management.create_account
+   ```
+
+1. Return to your Stage workspace. Click the **Add service** pop-up menu and select **Event**.
+
+   ![Click Add service in your workspace](../_images/add-event.png)
+
+1. On the **Add events** page, select your event provider, then click **Next**.
+
+   ![Select your event provider](../_images/download-workspace-config.png)
+
+1. Select the events to subscribe to. Then click **Next**.
+
+   ![Select the events to subscribe to](../_images/download-workspace-config.png)
+
+1. Optionally create a new JWT credential. Then click **Next**.
+
+1. Update the **Event registration name** and **Event registration description** fields. In the **How to receive events** section, under **Option 2**, select the runtime action you created in [Set up App Builder and define a runtime action](#set-up-app-builder-and-define-a-runtime-action).
```

Comment:
I was not able to see the runtime action created as part of the "Set up App Builder and define a runtime action" section in the dropdown here until I ran `aio app deploy`. Maybe that could be mentioned as a step either here or in that other section.

Reply:
Added a statement in line 94, except I mentioned the `run` command instead of `deploy`. That seems to work too.